src/main/java/BuyStockUseCase/BuyStockGUI.java

```diff
@@ -8,6 +8,9 @@
 import java.awt.*;
 
 public class BuyStockGUI extends JFrame implements iBuyStockGUI {
+    /**
+     * GUI where the user can purchase stocks
```

Comment:
I would recommend adding something simple about the class attributes such as the functionality of the buttons